logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/UpdateConversationReadDateUseCase.kt

```diff
@@ -37,7 +37,8 @@ class UpdateConversationReadDateUseCase internal constructor(
      */
     suspend operator fun invoke(conversationId: QualifiedID, time: Instant) {
         selfConversationIdProvider().flatMap { selfConversationIds ->
-            sendConfirmation(conversationId)
+            // TODO: Disabled for now as we are still figuring out performance and STORAGE_ERROR issues.
+            // sendConfirmation(conversationId)
             conversationRepository.updateConversationReadDate(conversationId, time.toIsoDateTimeString())
             selfConversationIds.foldToEitherWhileRight(Unit) { selfConversationId, _ ->
                 sendLastReadMessageToOtherClients(conversationId, selfConversationId, time)
```

Comment:
how about disabling the sendReadReceipt to the self aswell?

Reply:
As they are two different things I suppose we still want to keep updating the last time we read the conversation? 🤔

Reply:
so still we're sending messages related to the read-receipts [one less in this case :D ] but   still sending! so I my suggestion to cut all the connections with outside related to read-receipts.

Reply:
we want to update the last_read_date locally but don't send anything